0xOpenBytes/ios-base
# updated Swish package to include custom header logic

## Description

This PR edits the Swish project file to include a custom header logic. If a user clones the project but does not run the `./rename.sh` script, any new file generated will generate with our custom header to include the MIT licensing and generation language for OpenBytes. Alternatively, if a user does run the `./rename.sh` file the new header with their logic.

## Testing Steps

1. Pull changes.
2. Run `./rename.sh`.
3. Open new project in Xcode.
4. *Note* The default header should not be changed.
5. Create a new file.
6. New header should be displayed.

## Areas of Code Impacted

Swish Project -> Project.swift
`./rename.sh` rename of xcodegen to swish

## Other Notes/Considerations

N/A
